itou/templates/apply/process_refuse.html

```diff
@@ -102,6 +107,12 @@ <h2 class="mb-3 mb-md-4">Réponse au prescripteur</h2>
                                         <span class="c-info__summary">En choisissant ce motif, les fiches de postes associées seront dépubliées.</span>
                                     </div>
                                 </fieldset>
+                                <fieldset>
+                                    <div class="form-group">
+                                        <label>Partager ce motif de refus au candidat ?</label>
```

Comment:
Il ne faudrait pas utiliser une `<legend>` plutôt qu’un `<label>` pour ce texte ? Comme on a un `<fieldset>`.

Ou alors, mettre un `for="id_reason-refusal_reason_shared_with_job_seeker"` ?

Reply:
C'est pour être cohérent [avec le label](https://github.com/gip-inclusion/les-emplois/blame/f96328c5e1e84a1066e3116725bb7516fd454f6c/itou/templates/apply/process_refuse.html#L17) qui est utilisé au dessus pour la liste de raisons.
Un `<legend>` a un rendu beaucoup plus gros.

Je tag @hellodeloo [qui avait ajouté le fieldset](https://github.com/gip-inclusion/les-emplois/commit/a331ac26664cebbf91f3894536149044ad31d52c) pour validation ;)

Reply:
On pourrait ajouter un `<legend>` même si ce n'est pas obligatoire.
Effectivement, la mise en forme du `<legend>` (texte assez grand) est plutôt prévu pour les formulaires long avec plusieurs `<fielset>` + `<legend>` dedans. C'est pour ça que j'avais triché en mettant un `<label >` à la place. 
Dans ce cas, s'il y a `<legend>` il faudra ajouter les classes `.fs-base .font-weight-bold` pour simuler la mise en forme du `<label>`